metrotranscom/doorway
# fix: update header and remove how did you hear

# Pull Request Template

## Issue Overview

This PR addresses #457 


## Description

This PR updates the demographic header + subheader text along with removing the howDidYouHear section per QA notes. There are still outstanding questipns around the selection type for the other demographic questions.

NOTE: That there are many more references to how did you hear than what is changed here. Given this is a quick fix, I'd like to touch base with product that we never want this functionality in Doorway before clearing out all the strings, types, etc.

## How Can This Be Tested/Reviewed?

Ensure the demographics page matches the Revised Demographic Section of this [document](https://docs.google.com/document/d/12PbB8Ign93BiI1GOUAQrsc6DbNQ7IlRJjNXzaOd8hPY/edit#heading=h.z2we68wtdnff)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
